src/sink/http.go

```diff
@@ -17,7 +17,7 @@ const (
 	// DefaultRequestTimeout is the default IO timeout for each request.
 	DefaultRequestTimeout = 15 * time.Second
 	// DefaultAgentForwarderEndpoint holds the default endpoint of the agent forwarder.
-	DefaultAgentForwarderEndpoint = "http://localhost:8001/v1/data"
+	DefaultAgentForwarderEndpoint = "http://localhost:%s/v1/data"
```

Comment:
Might easily break with v6 addresses. Could we split this into two consts, e.g. `DefaultAgentForwarderAddress` and DefaultAgentForwarderPath`? That way we could use `net.JoinHostPort` which is v6-safe.